packages/babel-plugin-proposal-class-static-block/test/fixtures/integration-loose/super-static-block/output.js

```diff
@@ -1,26 +1,17 @@
-var _class, _2, _temp;
+var _class, _temp;
 
-var _ = /*#__PURE__*/babelHelpers.classPrivateFieldLooseKey("_");
-
-class Foo extends (_temp = (_2 = /*#__PURE__*/babelHelpers.classPrivateFieldLooseKey("_"), _class = class {}), Object.defineProperty(_class, _2, {
-  writable: true,
-  value: (() => {
-    _class.bar = 42;
-  })()
-}), _temp) {}
+class Foo extends (_temp = _class = class {}, (() => {
+  _class.bar = 42;
+})(), _temp) {}
 
 Foo.bar = 21;
-Object.defineProperty(Foo, _, {
-  writable: true,
-  value: (() => {
-    var _class2, _3, _temp2;
 
-    Foo.foo = (_temp2 = (_3 = /*#__PURE__*/babelHelpers.classPrivateFieldLooseKey("_"), _class2 = class {}), Object.defineProperty(_class2, _3, {
-      writable: true,
-      value: (() => {
-        _class2.bar = 42;
-      })()
-    }), _temp2).bar;
-  })()
-});
+(() => {
+  var _class2, _temp2;
+
+  Foo.foo = (_temp2 = _class2 = class {}, (() => {
+    _class2.bar = 42;
+  })(), _temp2).bar;
```

Comment:
I think this is a bug (the super class is evaluated twice), but the bug was present also before this PR.